connector/jobrunner/channels.py

```diff
@@ -632,12 +632,21 @@ def __init__(self):
         self._root_channel = Channel(name='root', parent=None, capacity=1)
         self._channels_by_name = WeakValueDictionary(root=self._root_channel)
 
+    @staticmethod
+    def split_strip(s, sep, maxsplit=-1):
+        """Split string and strip each component.
+
+        >>> ChannelManager.split_strip("foo: bar baz\\n: fred:", ":")
+        ['foo', 'bar baz', 'fred', '']
+        """
+        return [x.strip() for x in s.split(sep, maxsplit)]
```

Comment:
It was already like this on the 8.0-branch version of this change.

I could easily fix this here, but I'm afraid it could impact future changes done to multiple branches.

Should I:

 * fix it just here (and on the future branch 10 version of this PR)?
 * leave it as is?
  * And perhaps in the future do another PR for fixing this in the three branches?